crates/storage/src/column.rs

```diff
@@ -62,10 +62,8 @@ pub enum Column {
     // Below are the tables used for p2p, block production, starting the node.
     /// The column id of metadata about the blockchain
     Metadata = 17,
-    /// See `FuelBlockSecondaryKeyBlockHeights`
-    FuelBlockSecondaryKeyBlockHeights = 18,
     /// See [`SealedBlockConsensus`](crate::tables::SealedBlockConsensus)
-    FuelBlockConsensus = 19,
+    FuelBlockConsensus = 18,
```

Comment:
Since we don't have mainnet it's okay to break the numbering and invalidate the database. But yeah, later we will try to avoid breaking stuff like this